src/index.ts

```diff
@@ -224,6 +224,28 @@ function b32decodeXemAddr(data: string): Buffer {
   return nemSdk.default.utils.convert.ua2hex(nemSdk.default.model.address.b32decode(address));
 }
 
+function ontHexToBase58(hexEncoded: Buffer): string {
+  let addr: Buffer;
+  const ADDR_VERSION = '17';
```

Comment:
Can you please follow the pattern used for, eg, `makeBase58CheckEncoder` etc and take this as a parameter?

Reply:
Done it